actionlist/app/lib/commons.js

```diff
@@ -67,3 +67,22 @@ export const filterTypes = {
     title: 'Document Assigned Date',
   },
 };
+
+export const defaultState = {
```

Comment:
This object definitely belongs in the `actionItems` reducer and not in `commons`. I believe it can be imported in your test by using `import reducer, { defaultState } from 'path/to/actionItems'`.